src/olympia/addons/tests/test_views.py

```diff
@@ -2858,6 +2858,8 @@ def test_filter_by_guid_return_to_amo(self):
         addon_factory()
         self.reindex(Addon)
 
+        # need to keep this force_text because pre django2.2
+        # urlsafe_base64_encode returns a bytestring and a string after
```

Comment:
The comment is confusing (to me at least), maybe `need to keep force_text because urlsafe_base64_encode only starts returning a string from Django 2.2 onwards, before that a bytestring.` or so?

Reply:
https://github.com/mozilla/addons-server/pull/10700/commits/8abb7668faa2b989928a4821c662a16be507169d